### PR TITLE
Install BulkRAX

### LINKED ISSUE #811: Try the Hyrax batch ingest gem
### Descriptive summary

There's a hyrax batch ingest gem that may be a good community-supported alternative to our current CSV upload implementation.

Let's see if it can be easily integrated into Scholar so we can test out it's features.  We will then want to demo it for Content Services to see if we all agree that it's a worthy addition.

The gem is located https://github.com/samvera-labs/hyrax-batch_ingest

There are also slides and an audio recording from a Samvera Virtual Connect 2019 session where this was demonstrated.  Probably a good idea to review that for details that may be missing on the README.

### PR DESCRIPTION
**Pull request by scherztc**
Fixes #811 

Installs and configures the BulkRAX gem.
Modifies the application css to show Administrative Sets.

### CODE REVIEW

**Comment by scherztc** [comment]
https://wiki.lyrasis.org/display/samvera/2019-11-04+Meeting+notes
https://docs.google.com/presentation/d/1d7_vGFwrV-u2qC6l43tvLXWACP3dznhyhPcKdeX7rks/edit#slide=id.g65b791595c_0_0

BulkRAX is up on nurax.curationexperts.com.  I am requesting that my account be elevated to admin, so I can compare the functionality with this PR.

**Comment by scherztc** [comment]
It looks like we have 5 failing test in both local and CircleCi after rebase on develop.

rspec ./spec/features/hyrax/create_work_admin_spec.rb:40 # Creating a new Work as admin when there are multiple admin sets allows default admin set to be the first item in the select menu
rspec ./spec/features/hyrax/create_work_spec.rb:98 # Creating a new Work when the user is a proxy allows on-behalf-of deposit
rspec ./spec/features/hyrax/edit_content_block_admin_spec.rb:41 # Editing content blocks as admin when user wants to change tabs does not change tab when user dismisses the confirmation
rspec ./spec/features/hyrax/create_child_work_spec.rb:57 # Creating a new child Work when it's being updated can be updated
rspec ./spec/features/hyrax/create_child_work_spec.rb:63 # Creating a new child Work when it's being updated doesn't lose other memberships

I think these might be related to when we display the admin set and when we don't.  The Admin Set has to be displayed on the BulkRax dashboard but not any where else.

**Comment by scherztc** [comment]
ERROR: Default admin set exists but it does not have an associated permission template.

This may happen if you cleared your database but you did not clear out Fedora and Solr.

You could manually create the permission template in the rails console (non-destructive):

    Hyrax::PermissionTemplate.create!(source_id: AdminSet::DEFAULT_ID)

OR you could start fresh by clearing Fedora and Solr (destructive):

    require 'active_fedora/cleaner'
    ActiveFedora::Cleaner.clean!

**Comment by scherztc** [comment]
@hortongn :  I did a few skips on this test : spec/features/hyrax/create_child_work_spec.rb.

I am not sure why we have this test in there.  This test is copied over from hyrax and we made no changes to it.  There was already a skip in there around using the Fill In for Title stuff.

**Comment by hortongn** [comment]
closing and reopening this PR to try and make coveralls happy.

File: spec/features/hyrax/edit_work_spec.rb [hunk 1]
```diff
@@ -57,12 +57,6 @@
              agent_id: user.user_key)
     end
 
-    it 'selects admin set already assigned' do
```
[the remark this answers is not in the document]
Comment (scherztc) [reply spec/features/hyrax/edit_work_spec.rb:60]: Done.

File: Gemfile [hunk 2]
```diff
@@ -8,6 +8,7 @@ git_source(:github) do |repo_name|
 end
 
 gem 'browse-everything'
+gem 'bulkrax', git: 'https://github.com/samvera-labs/bulkrax.git'
```
[the remark this answers is not in the document]
Comment (scherztc) [reply Gemfile]: Pinned.

File: Gemfile [hunk 3]
```diff
@@ -115,3 +116,5 @@ end
 group :production do
   #  gem 'clamav'
 end
+
+gem 'willow_sword', github: 'notch8/willow_sword'
```
[the remark this answers is not in the document]
Comment (scherztc) [reply Gemfile]: Removed.  I think it has to do with Notch8's server environment.

File: spec/features/hyrax/create_child_work_spec.rb [hunk 4]
```diff
@@ -55,12 +55,14 @@
       parent.save!
     end
     it 'can be updated' do
+      skip
```
[the remark this answers is not in the document]
Comment (scherztc) [reply spec/features/hyrax/create_child_work_spec.rb]: I was able to get these testing passing by using the clean_repo and adding all the required fields to the create and edit test.

**Comment by hortongn** [comment]
@scherztc all looks good except for my suggestion to add a test for bulkrax.  See the text of my earlier review.

**Pull request merged**